src/components/layout/Footer.tsx

```diff
@@ -83,7 +83,7 @@ const Footer = () => (
               </Nav.Link>
             </Nav.Item>
           </Nav>
-          <p>&copy; Chicago Artist Guide 2021</p>
+          <p className="copy-text">&copy; Chicago Artist Guide 2021</p>
```

Comment:
`<CopyText>&copy; Chicago Artist Guide 2021</CopyText>`

Then below, instead of adding a class inside of `FooterNavbar`, you would just create your own styled component, like so:

```
const CopyText = styled.p`
  font-size: 9pt;

  ${media.smaller`
    font-size: 7pt;
  `}
`;
```

I'd also suggest using px, rem, or em instead of pt for font sizes.